src/Utilities/functions.jl

```diff
@@ -355,8 +355,7 @@ function scalar_terms_at_index(
     terms::Vector{<:Union{MOI.VectorAffineTerm,MOI.VectorQuadraticTerm}},
     i::Int,
 )
-    I = findall(t -> t.output_index == i, terms)
-    return map(i -> terms[i].scalar_term, I)
+    return [terms[idx].scalar_term for idx in eachindex(terms) if terms[idx].output_index == i]
```

Comment:
Wouldn't it be even faster to do `for term in terms if term.output_index == 1` ?

Reply:
right, I followed the initial structure too closely